dvs10_Options.m

```diff
@@ -0,0 +1,883 @@
+dashboard(3, sprintf('Performance: %0.0f%% correct',TrialRecord.User.perc_correct_trials), white)
+
+% save trial variables 
+if x_position1_i==1                                     % option 1 is on the left
+    dvs_left_value  = [dvs_left_value option1_value];
+    dvs_left_mag    = [dvs_left_mag mag1_i];
+    dvs_left_prob   = [dvs_left_prob prob1_i];
+    dvs_right_value = [dvs_right_value option2_value];
+    dvs_right_mag   = [dvs_right_mag mag2_i];
+    dvs_right_prob  = [dvs_right_prob prob2_i];
+elseif x_position1_i==2                                 % option 1 is on the right
+    dvs_left_value  = [dvs_left_value option2_value];
+    dvs_left_mag    = [dvs_left_mag mag2_i];
+    dvs_left_prob   = [dvs_left_prob prob2_i];
+    dvs_right_value = [dvs_right_value option1_value];
+    dvs_right_mag   = [dvs_right_mag mag1_i];
+    dvs_right_prob  = [dvs_right_prob prob1_i];
+end
+bhv_variable('dvs_left_value', dvs_left_value);
+bhv_variable('dvs_left_mag', dvs_left_mag);
+bhv_variable('dvs_left_prob', dvs_left_prob);
+bhv_variable('dvs_right_value', dvs_right_value);
+bhv_variable('dvs_right_mag', dvs_right_mag);
+bhv_variable('dvs_right_prob', dvs_right_prob);
+bhv_variable('stimulus_set',stimulus_set);
+
+%% ITI
+% start trial with blank screen = baseline/pre-fixation epoch
+set_bgcolor(bg_trial)
+eventmarker(ec_iti_start)
+idle(i ti)
+eventmarker(ec_iti_end)
+
+%% 1. FIXATION
+% need to fixate centre (fixation breaks allowed). if no fixation acquired
+% trial is stopped, error screen and timeout
+run_scene(fixation_scene,ec_fix_screen_shown); 
+
+    % no fixation
+    if ~fixth.Success
+            trialerror(ec_out_nofix)
+            idle(nofix_timeout,bg_nofix)
+            eventmarker(ec_timeout_screenshown)
+    end
+    
+
+%% 2. OPTION 1 ONSET
+% option 1 is shown. no fixation required to continue. record if/which cues
+% are fixated.
+if fixth.Success
+TrialRecord.User.num_initiated_trials = TrialRecord.User.num_initiated_trials+1;
+
+option1_onset_time = run_scene(option1_scene,ec_option1_on);
+  
+    if mag_fth1.Success
+        eventmarker(ec_mag1_fix)
+        if x_position1_i==1     % option 1 on the left
+            if mag1_y_position_i==1     % mag is up
+                eventmarker(ec_upleft_fix)
+            else
+                eventmarker(ec_downleft_fix)
+            end
+        else
+            if mag1_y_position_i==1    
+                eventmarker(ec_upright_fix)
+            else
+                eventmarker(ec_downright_fix)
+            end
+        end
+
+        time_mag1_acquired = mag_fth1.AcquiredTime;                              
+
+        % log RT and precise timings
+        rt_mag1 = time_mag1_acquired - option1_onset_time;
+        dvs_rts_mag1 = [dvs_rts_mag1 rt_mag1];
+        dvs_ts_mag1_acquired = [dvs_ts_mag1_acquired time_mag1_acquired];
+        dvs_mag1_held = [dvs_mag1_held fta_mag1.FixTime];
+        bhv_variable('dvs_rts_mag1', dvs_rts_mag1);

```

Comment:
nowadays it looks like you can just do `fth.RT`, which is probably safer than my janky code which you have inherited